cfme/containers/pod.py

```diff
@@ -57,6 +59,19 @@ def get_random_instances(cls, provider, count=1, appliance=None):
         return [cls(obj.name, obj.project_name, provider, appliance=appliance)
                 for obj in itertools.islice(pod_list, count)]
 
+    @staticmethod
```

Comment:
So, a few things here,
1) This function/method is dealing with pods from a specific provider
Therefore, either this function should be a method of the provider object, then there is no need to pass a provider to it.